data/IL/incentives.json

```diff
@@ -0,0 +1,1110 @@
+    "amount": {
+      "type": "dollars_per_unit",
+      "number": 250,
+      "unit": "ton"
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$250/ton rebate for qualifying Air Source Heat Pumps."
+    },
+    "start_date": 2024,
+    "end_date": 2024
+  },
+  {
+    "id": "IL-39",
+    "authority_type": "utility",
+    "authority": "il-jo-carroll-energy-cooperative",
+    "payment_methods": [
+      "account_credit"
+    ],
+    "item": "heat_pump_air_conditioner_heater",
+    "program": "il_jo-CarrollEnergyCooperative_energyEfficiencyIncentives",
+    "amount": {
+      "type": "dollars_per_unit",
+      "number": 250,
+      "unit": "ton"
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$250/ton rebate for qualifying Mini Split Heat Pumps."
+    },
+    "start_date": 2024,
+    "end_date": 2024
+  },
+  {
+    "id": "IL-40",
+    "authority_type": "utility",
+    "authority": "il-jo-carroll-energy-cooperative",
+    "payment_methods": [
+      "account_credit"
+    ],
+    "item": "weatherization",
+    "program": "il_jo-CarrollEnergyCooperative_energyEfficiencyIncentives",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 500,
+      "maximum": 500
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Up to $500 back on home insulation and air sealing."
+    },
+    "start_date": 2024,
+    "end_date": 2024
+  },
+  {
+    "id": "IL-41",
+    "authority_type": "utility",
+    "authority": "il-jo-carroll-energy-cooperative",
+    "payment_methods": [
+      "account_credit"
+    ],
+    "item": "electric_stove",
+    "program": "il_jo-CarrollEnergyCooperative_energyEfficiencyIncentives",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 25,
+      "maximum": 25
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$25 bill credit for any inductive range purchased and installed."
+    },
+    "start_date": 2024,
+    "end_date": 2024
+  },
+  {
+    "id": "IL-42",
+    "authority_type": "utility",
+    "authority": "il-jo-carroll-energy-cooperative",
+    "payment_methods": [
+      "account_credit"
+    ],
+    "item": "other",
+    "program": "il_jo-CarrollEnergyCooperative_energyEfficiencyIncentives",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 500,
+      "maximum": 500
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Up to $500 for Compressed Air Audit performed by Professional Engineer, Certified Energy Manager, or a cooperative pre-approved partner."
+    },
+    "start_date": 2024,
+    "end_date": 2024
+  },
+  {
+    "id": "IL-43",
+    "authority_type": "utility",
+    "authority": "il-jo-carroll-energy-cooperative",
+    "payment_methods": [
+      "account_credit"
+    ],
+    "item": "other",
```

Comment:
These have an item now – it's non_heat_pump_water_heater, so you should use that instead of other. You may have to edit or ignore the data validations in Google sheets. Same for some of the other `other`s.

Reply:
Where is the source of truth for these? Do we have one for energy audits as well?

Reply:
found it https://github.com/rewiringamerica/api.rewiringamerica.org/blob/main/src/data/types/items.ts

Reply:
@datu925 the remaining entities marked other should just be energy audits. It didn't look like we had a better classification for those yet.

Reply:
> @datu925 the remaining entities marked other should just be energy audits. It didn't look like we had a better classification for those yet.

For better or for worse, it looks like we lump energy audits under weatherization (see https://rewiringameri-g3x1100.slack.com/archives/C04E4THEGMT/p1708640580381249). Let's switch to that for now.